la_palma_overview/la_palma_overview_video.py

```diff
@@ -83,129 +94,170 @@ def make_video_from_images(path2night, video_path):
     Dependencies
     ------------
     Needs avconv from libav
-    """
+    '''
+    video_dir = os.path.dirname(video_path)
     avconv_command = [
         'avconv',
-        '-y', # force overwriting of existing output file 
-        '-framerate', '12', # 12 Frames per second
-        '-f', 'image2', 
-        '-i', os.path.join(path2night,'%06d.jpg'), 
+        '-y',                # force overwriting of existing output file
+        '-framerate', '12',  # 12 Frames per second
+        '-f', 'image2',
+        '-i',  os.path.join(image_path, '%06d.jpg'),
         '-c:v', 'h264',
-        '-s', '1920x1080', # sample images down to FullHD 1080p
-        '-crf', '23', # high quality 0 (best) to 53 (worst)
-        '-crf_max', '25', # worst quality allowed
+        '-s', '1920x1080',   # sample images down to FullHD 1080p
+        '-crf', '23',        # high quality 0 (best) to 53 (worst)
+        '-crf_max', '25',    # worst quality allowed
         video_path
     ]
-    avconv_stdout = open(os.path.join(path2night, 'avconv_stdout.txt'), 'w')
-    avconv_stderr = open(os.path.join(path2night, 'avconv_stderr.txt'), 'w')
+    avconv_stdout = open(os.path.join(video_dir, 'avconv_stdout.txt'), 'w')
+    avconv_stderr = open(os.path.join(video_dir, 'avconv_stderr.txt'), 'w')
     avconv_return_value = call(avconv_command, stdout=avconv_stdout, stderr=avconv_stderr)
     avconv_stdout.close()
     avconv_stderr.close()
     return avconv_return_value
 
-def logg_time(now):
-    return now.strftime("%Y %m %d %H:%M")
-
-class VideoStopWatch(object):
-    def __init__(self, path2night):
-        self.__start_video_convert = datetime.utcnow()
-        print(logg_time(self.__start_video_convert), "Create video from images in", path2night)
-    
-    def stop(self):
-        end_video_convert = datetime.utcnow()
-        time_to_convert = end_video_convert - self.__start_video_convert
-        print(logg_time(end_video_convert), "Video is done. Took", time_to_convert.seconds, "seconds to convert.")
-                       
-
-def la_palma_overview_video(output_path=None, working_path=None, trash_images=False):
-    """
+
+def date_path(date, base='', subdir=''):
+    return os.path.join(
+        base,
+        str(date.year),
+        str(date.month),
+        str(date.day),
```

Comment:
This functions just handles a given date, further down I create a suitable date.